### PR TITLE
feat(tileView): Display 3 participants in 1 row

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

**Review by damencho — APPROVED** [review]

**Comment by damencho** [comment]
The failure is due to the changes, the remote menu is not visible or something.

File: react/features/video-layout/functions.js [hunk 1]
```diff
@@ -110,7 +110,9 @@ export function getTileViewGridDimensions(state: Object) {
         - (iAmRecorder ? 1 : 0)
         - (disableSelfView ? 1 : 0);
 
-    const columnsToMaintainASquare = Math.ceil(Math.sqrt(numberOfParticipants));
+    // When there are 3 participants in the call we want them to be placed on a single row unless the maxColumn setting
+    // is lower.
+    const columnsToMaintainASquare = numberOfParticipants === 3 ? 3 : Math.ceil(Math.sqrt(numberOfParticipants));
```
Comment (saghul) [inline react/features/video-layout/functions.js:115]: Nit: can we make that number 3 a constant?

Comment (hristoterezov) [reply react/features/video-layout/functions.js:115]: This is mostly an exception. What would be the benefit of having constant?

**Comment by robertpin** [comment]
IIRC we didn't just want to show 3 people on one row, but even more, depending on the size of the client.

File: react/features/video-layout/functions.js (hunk 1 above)
Comment (saghul) [reply react/features/video-layout/functions.js]: Readability, nothing special.

**Comment by hristoterezov** [comment]
> IIRC we didn't just want to show 3 people on one row, but even more, depending on the size of the client.

@robertpin Good idea! I will discuss it with Emil! But lets leave it for version 2.

**Review by damencho — APPROVED** [review]

**Pull request merged**